core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -67,10 +67,14 @@ object AddPartitionsToTxnManager {
  *    genericErrorSupported: This maps to the case when the clients are updated to handle the TransactionAbortableException
  *    addPartition:          This allows the partition to be added to the transactions inflight with the Produce and TxnOffsetCommit requests. Plus the behaviors in genericErrorSupported.
  */
-sealed trait TransactionSupportedOperation
+sealed trait TransactionSupportedOperation {
+  val isTV2 = false;
```

Comment:
Should we name it `atLeastTV2` (if we have more protocol bumps they will inherit this override), or maybe `supportsEpochBump` to be consistent with TC naming (which tries to express the same thing).

Reply:
Hmm I think it is valid to rename. Let me think on supportsEpochBump. It might be the best option.